grabserial

```diff
@@ -207,6 +209,14 @@ verbose = 0         # pylint: disable=I0011,C0103
 cmdinput = u""      # pylint: disable=I0011,C0103
 
 
+
+def str_to_bool(input):
```

Comment:
I'd rather use 'value' here, as 'input' is a reserved word in some versions of python.